docs/cli-lib/api/getAndLoadConfigIfNeeded.md

```diff
@@ -0,0 +1,35 @@
+## `getAndLoadConfigIfNeeded(options)`
```

Comment:
The formatting was heavily inspired by [Redux](https://github.com/reduxjs/redux/blob/master/docs/api/applyMiddleware.md#applymiddlewaremiddleware)